### ISSUE TITLE
LARMOR: Adapt & EPICS-enable the current LabVIEW PLC VI

### ISSUE THREAD

**Issue by kjwoodsISIS** [ISISComputingGroup/IBEX#203]
As a support person, I want to adapt and EPICS-enable the existing LabVIEW VI for PLCs (as currently used on SANS2D), so that I can re-use it on LARMOR.

'''Acceptance Criteria'''
1. A LabVIEW VI to control the LARMOR PLC
1. EPICS-enablement of the LabVIEW VI
1. A CSS-based UI to view and change the state of the PLC

'''Notes'''
1. None at present

**Comment by kjwoodsISIS** [comment]
_Imported from trac issue 205.  Created by skn09965 on 2013-09-23T17:07:30, last modified: 2014-05-13T16:16:41_

**Comment by kjwoodsISIS** [comment]
_Trac comment by kvlb23 on 2013-10-18 11:23:44:_

This is unlikely to be required as the IOC from Diamond should be enough of a base to make the work to generate an appropriate IOC achievable.

**Comment by kjwoodsISIS** [comment]
_Trac comment by skn09965 on 2013-10-24 10:30:13:_

This is a low priority story.  Discuss with KVLB before starting this story.

**Comment by kjwoodsISIS** [comment]
_Trac comment by skn09965 on 2013-11-21 15:12:31:_

Postponed - may not be needed.

**Comment by kjwoodsISIS** [comment]
Hasn't been needed for almost 3 years!  Close this issue.  We can create a new ticket in the unlikely event the notion gets resurrected.
